### ISSUE TITLE
Use dark-grey circle icon for snapcraft

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical-websites/docs.ubuntu.com#147]
## Summary
The snapcraft logo is the only one outside a circle. There is a version of the logo in a circle, which would be more appropriate here. 
## Process

https://docs.ubuntu.com/

## Current and expected result

[describe what happened and what you expected]

## Screenshot

[if relevant, include a screenshot]

**Comment by anthonydillon** [comment]
@lyubomir-popov would you mind providing a link to the asset on the asset server please?

**Comment by lyubomir-popov** [comment]
@anthonydillon yes, just created this issue during a meeting and assignd myself to remember to prepare one. Will do tomorrow.

**Comment by lyubomir-popov** [comment]
@anthonydillon https://assets.ubuntu.com/v1/25302166-snapcraft-primary-icon--dark.svg

**Comment by lyubomir-popov** [comment]
The asset is ready, I'm unassigning myself.
